.github/workflows/ossf-scorecard.yml

```
@@ -4,24 +4,26 @@ name: OSSF Scorecard
 on:
   # For Branch-Protection check. Only the default branch is supported. See https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection.
   branch_protection_rule:
-  push:
-    branches: [main]
   # To guarantee Maintained check is occasionally updated. See https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained.
   schedule:
     - cron: "0 0 * * 1"
+  push:
+    branches: [main, master]
```

```suggestion
    branches: [main]
```
We can keep this to main since there is no master branch in this repo